app/controllers/session_displays_controller.rb

```diff
@@ -4,9 +4,10 @@
 # A controller for disabling session information
 class SessionDisplaysController < ApplicationController
   ##
-  # Update the session, to disable_search_context
+  # Update the session, to disable_search_context or disable_date_sort_context
   def update
-    session[:disable_search_context] = true
+    session[:disable_search_context] = true if params[:context] == 'result_info'
```

Comment:
Is it worth trying to align the session key with the `params[:context]` value?

Reply:
makes sense to me